source/install/troubleshooting.rst

```diff
@@ -629,3 +629,293 @@ To help us narrow down whether it’s a server configuration issue, device speci
+
+General Guidelines for Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow these guidelines when providing diagnostic data to us:
+
+* Make sure the files you provide are as complete as possible, rather than providing a few lines. Entire log files and configurations provide us with important context.
+* Provide configuration and log files in plaintext format if possible, as these are far easier for us to search than screenshots.
+* Be sure to sanitize configuration and log files to remove usernames, passwords, and LDAP groups. Replace these details with example strings that contain the same special characters if possible, as special characters are common causes of configuration errors.
+* Provide screenshots or screen recordings of unexpected product behavior so that we know exactly what your users are seeing.
+
+Configuration
+~~~~~~~~~~~~~
+
+Why we need your configuration data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On Linux systems, settings are generally stored in configuration files. Many issues can be resolved by enabling or disabling a configuration setting. In order to find a resolution, we need to have as complete a picture of your system setup as possible. This also helps us to reproduce bugs so our developers can fix them.
+
+What configuration data includes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Configuration includes (but is not limited to):
+
+- The Mattermost ``config.json`` file.
+- The configuration for the reverse proxy, e.g. Nginx, HAProxy, AWS.
+- The database configuration.
+- SAML configuration when the issue is regarding SAML authentication. The configuration for the Mattermost service is in the SAML IdP.
+- Any other systems that Mattermost connects to or systems that exist between the user and the Mattermost server.
+
+How to access your configuration data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Mattermost configuration**
+
+The Mattermost configuration is usually stored at ``/opt/mattermost/config/config.json``. If you've migrated the Mattermost configuration to the database, you can get the configuration using ``mmctl`` or by running this database query:
+
+.. code-block:: none
+    
+   SELECT Value FROM Configurations WHERE Active = 1;
+
+**Reverse Proxy configuration**
+
+Nginx usually splits its configuration into two parts: the main server configuration at ``/etc/nginx/nginx.conf``, and a virtual server configuration. On Ubuntu, this is stored in ``/etc/nginx/sites-available``. Providing both of these configuration files is helpful, but providing the latter is more important.
+
+**SAML configuration**
+
+If the issue you're seeing is with SAML login, we will need to see the full configuration for the Mattermost service in the SAML provider. Providing screenshots similar to the ones in the setup documentation is sufficient because most SAML providers are configured using a web interface.
```

Comment:
```suggestion
If the issue you're seeing is with SAML login, we will need to see the full configuration for the Mattermost service in the SAML provider. The configuration for the Mattermost service is in the SAML IdP. Providing screenshots similar to the ones in the setup documentation is sufficient because most SAML providers are configured using a web interface.
```